kafka/templates/log4j.properties.j2

```diff
@@ -15,47 +15,35 @@
 
 kafka.logs.dir=/logs
 
-log4j.rootLogger=INFO, stdout
+log4j.rootLogger={{ SERVER_LOG_LEVEL | default("INFO") }}, kafkaAppender
```

Comment:
do you have to default here since you did that above in the exports? same as the below?